tests/util/ConverterTestCase.py

```diff
@@ -23,9 +23,33 @@ class ConverterTestCase(usdex.test.TestCase):
         omni.asset_validator.IssuePredicates.ContainsMessage("ArticulationRootAPI definition on a kinematic rigid body is not allowed"),
     ]
 
-    def assert_rotation_almost_equal(self, rot1: Gf.Rotation, rot2: Gf.Rotation, tolerance: float = 1e-6):
-        self.assertTrue(Gf.IsClose(rot1.GetAxis(), rot2.GetAxis(), tolerance), f"Axis mismatch: {rot1.GetAxis()} != {rot2.GetAxis()}")
-        self.assertTrue(Gf.IsClose(rot1.GetAngle(), rot2.GetAngle(), tolerance), f"Angle mismatch: {rot1.GetAngle()} != {rot2.GetAngle()}")
+    def assert_rotation_almost_equal(self, rot1: Gf.Rotation | Gf.Quatf | Gf.Quatd, rot2: Gf.Rotation | Gf.Quatf | Gf.Quatd, tolerance: float = 1e-6):
```

Comment:
I would be ok to move this to usdex directly & put out 2.2.0a1 for you to use in this PR

Reply:
https://pypi.org/project/usd-exchange/2.2.0a1/